brownie-config.yml

```diff
@@ -23,3 +23,7 @@ reports:
         - IERC20
         - ReentrancyGuard
         - Address
+
+
+hypothesis:
+  max_examples: 100
```

Comment:
https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.max_examples

If `run_fuzz.py` is not included in test suite by default, maybe a bit larger number of `max_examples` could be used here, but of course it will take longer to complete :(

Reply:
100 takes a few seconds but 1000 takes a while, not sure if we want to enforce more